### PR TITLE
Fix pgen

### PR DESCRIPTION
**Pull request by gbryant-arm**
Change short name for --enable-clock to avoid collisions with other arguments (certificates).
Fixes crash when running pgen aka generate-policy

### CODE REVIEW

**Review by dominic-mulligan-arm — COMMENTED** [review]
Has the `test-collateral` Makefile, and anything else that may be calling `pgen`, been checked and updated to use the new flag, as appropriate?

**Comment by gbryant-arm** [comment]
Nothing seems to call `pgen` with short arguments.
I haven't run the tests, will wait for the CI.

**Comment by dominic-mulligan-arm** [comment]
This is failing due to `openssl-sys` updating earlier today and bumping the MSRV.  You will need to wait until #266 is merged.

**Review by dominic-mulligan-arm — APPROVED** [review]

**Pull request merged**
